testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/broker/AbstractBrokerTest.java

```diff
@@ -157,23 +174,33 @@ public void logInAsUserInIDP() {
     protected void testSingleLogout() {
         log.debug("Testing single log out");
 
-        driver.navigate().to(getAuthRoot() + "/auth/realms/" + providerRealmName() + "/account");
+        driver.navigate().to(getAccountUrl(providerRealmName()));
 
         Assert.assertTrue("Should be logged in the account page", driver.getTitle().endsWith("Account Management"));
 
+        String encodedAccount;
+        try {
+            encodedAccount = URLEncoder.encode(getAccountUrl(providerRealmName()), "UTF-8");
+        } catch (UnsupportedEncodingException e) {
+            encodedAccount = getAccountUrl(providerRealmName());
+        }
+
         driver.navigate().to(getAuthRoot()
                 + "/auth/realms/" + providerRealmName()
                 + "/protocol/" + "openid-connect"
-                + "/logout");
+                + "/logout?redirect_uri=" + encodedAccount);
 
-        driver.navigate().to(getAuthRoot() + "/auth/realms/" + providerRealmName() + "/account");
+        waitForPage("log in to " + providerRealmName());
```

Comment:
This shouldn't be necessary as the page is not an ajax page. Once driver.navigate is complete the page should be loaded.